### PR TITLE
Text alignment and global attributes

### PR DESCRIPTION
**Pull request by libertymayc**

### CODE REVIEW

File: packages/theme/css/global.css [hunk 1]
```diff
@@ -5,16 +5,15 @@
  * It would be nice to do these as a global reset, so each component doesn't need to set these.
  * - font-family
  * - line-height
- * - background
  * - color
  */
 
-p {
+ .uitk-theme {
```
Comment (libertymayc) [inline packages/theme/css/global.css:11]: @joshwooding this came up in a 101, but would be good to further discuss this or get your opinion

File: .changeset/neat-impalas-sort.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+"@jpmorganchase/uitk-core": minor
+"@jpmorganchase/uitk-lab": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+Make theme global font size `--uitk-text-fontSize` instead of 14px, and move to global.css from typography.css
+Give global values for font-size, font-family, color, line-height, letter-spacing
+
+Remove base 1.3 line height, replace with appropriate line height variant throughout code
+Remoce text background on default and hover state tokens
```
Comment (joshwooding) [inline .changeset/neat-impalas-sort.md:11]:
```suggestion
Remove text background on default and hover state tokens
```

File: .changeset/neat-impalas-sort.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+"@jpmorganchase/uitk-core": minor
+"@jpmorganchase/uitk-lab": minor
+"@jpmorganchase/uitk-theme": minor
+---
+
+Make theme global font size `--uitk-text-fontSize` instead of 14px, and move to global.css from typography.css
+Give global values for font-size, font-family, color, line-height, letter-spacing
+
+Remove base 1.3 line height, replace with appropriate line height variant throughout code
```
Comment (joshwooding) [inline .changeset/neat-impalas-sort.md:10]:
```suggestion
Remove base 1.3 line height, and replace with appropriate line height variant throughout the code
```

**Review by joshwooding — CHANGES_REQUESTED** [review]

**Review by joshwooding — APPROVED** [review]

**Comment by joshwooding** [comment]
There are some minor line-height changes in the chromatic changes I would like a designer to look at

**Review by joshwooding — APPROVED** [review]

**Pull request merged**